### DISCUSSION TITLE
why unbable to breakout

### DISCUSSION THREAD

**Discussion by bingcnqz** [Tecnativa/doodba#641] [Q&A]
can someone help to guid how to debug under windows wsl env?

![image](https://github.com/user-attachments/assets/9f41e1da-fc27-4f16-8c77-568dc88b3d9e)

![image](https://github.com/user-attachments/assets/82c48065-6b17-460e-9e94-28d352a0ba95)

**Comment by bingcnqz** [comment]
![image](https://github.com/user-attachments/assets/8297983c-8a82-44e8-b04b-f4f4b4ac8ee3)
it can not hit

**Comment by pedrobaeza** [comment]
AFAIK, you can't do it in Windows because Windows doesn't support file links, and Doodba are using them for linking dynamically the addons in the `auto` folder. I recommend you to go to Linux.

**Reply by bingcnqz** [reply]
thanks for your info 
i am going to  try it in vm ubuntu then

**Reply by bingcnqz** [reply]
![Screenshot from 2024-12-31 12-29-56](https://github.com/user-attachments/assets/7275ff98-49b9-43c8-ad52-c25a61996631)
still unable to in ubuntu 2204 
is there any guide on how to debug pls?
thanks

**Reply by pedrobaeza** [reply]
Are you working with https://github.com/Tecnativa/doodba-copier-template/ in development mode?

Check https://github.com/Tecnativa/doodba-copier-template/blob/main/docs/daily-usage.md

**Reply by bingcnqz** [reply]
> Are you working with https://github.com/Tecnativa/doodba-copier-template/ in development mode?
> 
> Check https://github.com/Tecnativa/doodba-copier-template/blob/main/docs/daily-usage.md

yes 
[doodba-copier-template/](https://github.com/Tecnativa/doodba-copier-template/) in development mode

follow full steps but still problem  :not able to breakout
 
after read https://github.com/Tecnativa/doodba-copier-template/blob/main/docs/daily-usage.md#visual-studio-code-integration
and https://github.com/Tecnativa/doodba-copier-template/blob/main/docs/faq.md
